### DISCUSSION TITLE
Error buidling site with islands on Windows using git-bash

### DISCUSSION THREAD

**Discussion by davidlueder** [ElMassimo/iles#107] [Q&A] [answered]
Hello,
i might have found a strange bug while building a project with partial hydration. On my machine all builds in git-bash are failing, but building it within a wsl container does work.

### Error
```
Error: Could not resolve entry module (index_ile-2).
    at error (D:\projekte\iles-test-windows-bug\node_modules\rollup\dist\shared\rollup.js:160:30)
    at ModuleLoader.loadEntryModule (D:\projekte\iles-test-windows-bug\node_modules\rollup\dist\shared\rollup.js:22413:20)
    at async Promise.all (index 1) {
  code: 'UNRESOLVED_ENTRY',
  watchFiles: [ 'D:/projekte/iles-test-windows-bug/src/components/Welcome.vue' ]
}
```

### My Versions
**OS:** Windows 10 Pro, Build 19044.1586
**Git:** 2.30.0.windows.2
**Node:** v16.14.2
**NPM:** 8.5.5
**WSL:** Ubuntu Server 20.04 LTS (same node + npm versions)

### Steps to reproduce
1. Download full project: https://stackblitz.com/edit/iles-31wppn?file=src%2Fpages%2Findex.mdx (added "client:load" to `<Welcome/>`)
2. Extract archive
3. Open Project in Git Bash
4. npm install
5. npm run build

**>> Build failed**
![image](https://user-images.githubusercontent.com/16711294/159812957-23ce5310-6cbb-4c35-8444-dca1ba85b932.png)

6. Open Project in WSL
7. npm install
8. npm run build

**>> Build successful**
![image](https://user-images.githubusercontent.com/16711294/159813079-28462d1c-9cf1-4614-b535-ad9ed3c0dd72.png)

Can anyone try to reproduce this on a windows machine?

I'm wondering if the file paths have anything to do with this...

**Comment by ElMassimo** [comment]
Hi David!

Given it works in WSL, it's very likely that it's related with file paths.

Perhaps you can try upgrading `iles` (the example uses `0.7.27`), `vite` (the example uses a beta version), and other dependencies to the latest versions, in case the problem has already been fixed.

--- 

Internally, îles and its modules use [`pathe`](https://github.com/unjs/pathe), which helps to prevent many incompatibilities.

A friend regularly builds a site on Windows using git-bash, and he hasn't run into this problem. I might ask him to download the link you provided and see if we can replicate it.

In the meantime, if using WSL is an option, go for it 😃

**Comment by luminarious** [comment]
Same issue appears using Powershell on Windows 11 with Iles 0.7.36

**Comment by ElMassimo** [comment, marked as answer]
@davidlueder Thanks for reporting and providing a trace!

Fixed in [`iles@0.7.37`](https://github.com/ElMassimo/iles/blob/main/packages/iles/CHANGELOG.md#0737-2022-04-03)
